packages/uniforms-antd/src/ListAddField.tsx

```diff
@@ -33,10 +28,11 @@ function ListAdd({
 }: ListAddFieldProps) {
   const nameParts = joinName(null, name);
   const parentName = joinName(nameParts.slice(0, -1));
-  const parent = useField<
-    { initialCount?: number; maxCount?: number },
-    unknown[]
-  >(parentName, { initialCount }, { absoluteName: true })[0];
+  const parent = useField<{ maxCount?: number }, unknown[]>(
+    parentName,
```

Comment:
I guess we could inline `parentName` now? It won't be _as awfully_ formatted as before.

Reply:
Not sure, don't have a strong preference because I don't mind the current approach. It's longer but explicit and easy to read, whereas if I inline all of it it would be much less readable IMO.